packages/@react-aria/color/src/useColorSlider.ts

```diff
@@ -61,6 +62,32 @@ export function useColorSlider(props: ColorSliderAriaOptions, state: ColorSlider
     inputRef
   }, state);
 
+  let {step, pageSize} = state.value.getChannelRange(channel);
+  let {keyboardProps} = useKeyboard({
+    onKeyDown(e) {
+      // these are the cases that useMove or useSlider don't handle
+      if (!/^(PageUp|PageDown)$/.test(e.key)) {
+        e.continuePropagation();
+        return;
+      }
+      // same handling as useMove, stopPropagation to prevent useSlider from handling the event as well.
+      e.preventDefault();
+      let pageStep = Math.max(pageSize, step);
+      // remember to set this so that onChangeEnd is fired
+      state.setThumbDragging(0, true);
+      switch (e.key) {
+        case 'PageUp':
+          state.incrementThumb(0, pageStep);
+          break;
+        case 'PageDown':
+          state.decrementThumb(0, pageStep);
+          break;
+      }
+      // wait a frame to ensure value has changed then unset this so that onChangeEnd is fired
+      state.setThumbDragging(0, false);
+    }
+  });
```

Comment:
Doesn't useSliderThumb handle this? I saw the above discussion but I'm not sure why it is resolved?

Reply:
I had readded it cuz the behavior broke with the removal of pageSize from useSliderState props in https://github.com/adobe/react-spectrum/pull/2917, but I can re-remove it since we want to keep pageSize prop in useSliderState